analysis/org.eclipse.tracecompass.analysis.chromium.core/src/org/eclipse/tracecompass/internal/analysis/chromium/core/callstack/ChromiumCallStackProvider.java

```diff
@@ -92,20 +92,30 @@ protected boolean considerEvent(@NonNull ITmfEvent event) {
 
     @Override
     protected @Nullable ITmfStateValue functionEntry(@NonNull ITmfEvent event) {
-        if (event instanceof TraceEventEvent && 'B' == (((TraceEventEvent) event).getField().getPhase())) {
+        if (event instanceof TraceEventEvent && isEntry(event)) {
             return TmfStateValue.newValueString(event.getName());
         }
         return null;
     }
 
+    private static boolean isEntry(ITmfEvent event) {
+        char phase = ((TraceEventEvent) event).getField().getPhase();
```

Comment:
Updated, the patch, I think in 2 weeks, (Don't want to wear out my colleagues) the open source stuff may pop up on gerrit.

Also, since when do we have job descriptions?!?